.travis.yml

```diff
@@ -7,33 +7,52 @@ matrix:
   include:
     - php: 7.1
       sudo: false
-      env: SYMFONY_VERSION=3.4.* PHPSTAN=true
+      env: SYMFONY_VERSION=^3.4 PHPSTAN=true
     - php: 7.1
       sudo: false
-      env: SYMFONY_VERSION=3.4.* PHP_CS_FIXER=true
+      env: SYMFONY_VERSION=^3.4 PHP_CS_FIXER=true
     - php: 7.1
       sudo: false
-      env: SYMFONY_VERSION=3.4.* UNIT_TESTS=true
+      env: SYMFONY_VERSION=^3.4 UNIT_TESTS=true
     - php: 7.1
       sudo: false
-      env: SYMFONY_VERSION=4.0.* UNIT_TESTS=true
+      env: SYMFONY_VERSION=^4.0 UNIT_TESTS=true
     - php: 7.2
       sudo: false
-      env: SYMFONY_VERSION=4.0.* UNIT_TESTS=true
+      env: SYMFONY_VERSION=^4.0 UNIT_TESTS=true
+    - php: 7.3
+      sudo: false
+      env: SYMFONY_VERSION=^4.0 UNIT_TESTS=true
     - php: 7.1
       services: docker
       sudo: required
-      env: SYMFONY_VERSION=3.4.* FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
+      env: SYMFONY_VERSION=^3.4 FUNCTIONAL_TESTS=true PREPARE_CONTAINER=true
```

Comment:
The version should be `x.y.*` format to ensure that only that version is allowed. Composer could install anew newer version for `^3.4` constraint.